testing/apt-dater-host/APKBUILD

```diff
@@ -3,23 +3,32 @@
 pkgname=apt-dater-host
 pkgver=1.0.0
```

Comment:
The source is not version 1.0.0, please don’t lie to Alpine users. If you really need to use unreleased revision, then add appropriate suffix to the version number, e.g. `1.0.0_git20170216`.